templates/jlink/README.md

```diff
@@ -1,58 +1,67 @@
-# OpenShift-JLink PoC
+# OpenShift Jlink integration (Tech Preview)
 
-Try it out:
+To try it out,
+you need:
 
-## Stage 1: create ubi9 jlink imagestream and push a ubi9 image with jmods installed to it.
+1. Access to an OpenShift instance, such as [OpenShift Local](https://developers.redhat.com/products/openshift-local/overview)
+2. UBI9 OpenJDK ImageStreams that include `jlink-dev` changes (see below)
+3. The template [jlinked-app.yaml](jlinked-app.yaml).
 
-You need:
+## Stage 0: UBI9 OpenJDK ImageStreams with jlink-dev changes
 
-1. Access to an OpenShift instance, such as crc
-2. UBI9 OpenJDK ImageStreams. You can load them from this repository with
+Until the `jlink-dev` work is merged, prior to trying out the template, we must first
+prepare UBI9 OpenJDK ImageStreams with `jlink-dev` support.
 
-    oc create -f templates/ubi9-community-image-streams.json
+1. Build a suitable OpenJDK container image from [this
+   repository](https://github.com/jboss-container-images/openjdk),
+   branch `jlink-dev`. e.g.
 
-Steps to produce the imagestream and image:
+        cekit --descriptor ubi9-openjdk-17.yaml build docker
 
-1. install the template
+2. Within your OpenShift project,
 
-    oc create -f templates/jlink/jlink-builder/jlink-builder-template.yaml
+        oc create imagestream ubi9-openjdk-17
 
-This will create a Template called jlink-builder-template, you should see
+3. Log into the OpenShift registry, e.g.
 
-    template.template.openshift.io/jlink-builder-template created
+        REGISTRY_AUTH_PREFERENCE=docker oc registry login
 
-and after running oc get template, it should be in the list as
+4. tag and push the dev image into it. The OpenShift console gives you the
+   exact URI for your instance
 
-    jlink-builder-template                        Template to produce an imagestream and buildconfig for a Jlink builder image       1 (all set)       2
+        docker tag ubi9/openjdk-17:1.18 default-route-openshift-image-registry.apps-crc.testing/jlink1/ubi9-openjdk-17:1.18
```

Comment:
After this step whether it's good to check whether we have :
$ cat /etc/docker/daemon.json 
	{
		"insecure-registries": [ "default-route-openshift-image-registry.apps-crc.testing" ]
	}
If not then create this and 

$ systemctl restart docker 
?

Other than this this PR looks fine to me.

Reply:
That's a good suggestion, thank you, I'll add that.